File: forum-client/src/Types/Reply.ts
export interface Reply{
    id: Number,
    text: string,
    date_Created: Date,
    date_Edited?: Date,
    comment_Id: Number,
    reply_Id: Number
    user_Id: Number,
    user_Username: string,
    reply_Username: string
    likes: Number,
    liked: boolean,
    is_Deleted: boolean
}
export interface ReplyInput {
    text: string,
    user_Id: Number,
    comment_Id: Number,
    reply_Id?: Number,
}

File: Domain/Entities/Reply.cs
using Domain.Constants;
using System;

namespace Domain.Entities
{
    public class Reply
    {
        public int Id { get; set; }
        public string Text { get; set; }
        public DateTime Date_Created { get; set; }
        public DateTime? Date_Edited { get; set; }
        public int Comment_Id { get; set; }
        public int? Reply_Id { get; set; }
        public int User_Id { get; set; }
        public Reply() { }
    }
}
